code/lib/test/package.json

```diff
@@ -49,8 +49,8 @@
     "@testing-library/dom": "10.4.0",
     "@testing-library/jest-dom": "6.5.0",
     "@testing-library/user-event": "14.5.2",
-    "@vitest/expect": "2.0.5",
```

Comment:
why was this change needed for writing E2E tests? This can have impact on users, do you what that impact would be?

Reply:
I ran into a problem when running the E2E tests, caused by multiple Vitest versions _somewhere_, can't remember where or what the problem was, but it was fixed by aligning all the versions.

I can't think of how this would affect users negatively, we upgrade non-breaking dependencies all the time and I'm not sure how this is different. It was just an attempt to do a bit of house cleaning while touching that code anyway.

But I don't care much about it, I'll revert.